src/Mailer/Email.php

```diff
@@ -2092,11 +2092,11 @@ protected function _logDelivery($contents)
     
     
     /**
-     * Converts $array to string if array given
-     * @param null $key
+     * Converts array to string
+     * @param array $array
```

Comment:
There is still no string annotation. I also wouldnt have used $array as variable name, it is a lie if it can also be a string.
$content is maybe more appropriate.

Reply:
$content is perfect.
why is phpcs failing is it due to some sort of hidden character, space or break?

Reply:
I guess that is failing due to PSR right?